src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function(a) {
+    count++;
```

Comment:
It's not a good place for incrementing the counter. Let's put it inside inner function